### ISSUE TITLE
Online demo

### ISSUE THREAD

**Issue by hadisfr** [aurora/flowdesigner#1]
What do you think about adding an online demo based on the provided [example](https://github.com/aurora/flowdesigner/tree/master/example) using the github-pages? 🤔

**Comment by aurora** [comment]
Yes, that would be really nice and i also have this on my wish list. Unfortunately, i didn't have time for it, so far 😞 ... maybe later, this year.

**Comment by hadisfr** [comment]
I may give it a try later. If it's successful, I'll make a pull request.

**Comment by aurora** [comment]
Sure, thanks for working on it!

**Comment by hadisfr** [comment]
I think it's somehow hard to create a pull request because I rewrite history.

By the way, there's a very simple online demo based on [example](https://github.com/aurora/flowdesigner/tree/master/example) available at <https://hadisfr.github.io/flowdesigner> now.
I think the simplest way is that you pull my [gh-pages](https://github.com/hadisfr/flowdesigner/tree/gh-pages) branch and push it to the same branch on your repo, and then select that branch from _Settings_ as _Source_ of _GitHub Pages_.

**Comment by aurora** [comment]
Mmmm ... that's strange, the example on your page seems to have some issues i thought i already resolved in the past, like the issue with the too short line when wiring up the nodes (the end of the red line is not below the mouse-pointer) and the issue where sometimes the node is dragged when you click the output bullet for drawing a wire. Strange ...

**Comment by hadisfr** [comment]
I didn't encountered the second one, but I have some guesses. 🤔 The first one occurs on the example, too. It was seemed worse on my gh page because I added a `max-width: 100%;` to canvas, which is removed now. Perhaps it fixes the second issue, too.
Also, It seems that the red line starts from the middle of starting connector but prolongates only until the borders of ending connector. It seems strange to me.

**Comment by aurora** [comment]
Ah, so it was the max-width setting, that caused the issues - yes the second issue is gone now, too. But i see what you mean with the ending connector. This is probably a bug in the code and should be fixed.

**Comment by hadisfr** [comment]
Let's provide a screenshot for documentation. :D
![image](https://user-images.githubusercontent.com/24562441/44468809-4d011d00-a63b-11e8-8502-18b68e13f539.png)
I tested in on firefox, chrome, and safari, and it's the same.

**Comment by aurora** [comment]
Yes, i think it shouldn't be that hard to fix. I'll have a look at it at the weekend.

**Comment by hadisfr** [comment]
It's something related to `calcLine`

https://github.com/aurora/flowdesigner/blob/1c9da16e1ffdc0f1b2c2b313c3d3dfc11737d819/src/wire.js#L29-L49

There is a small space between the pointer and the end of the path, too. I don't get what this function is expected to do actually, (creating this space?) but it seems there's no need to this function in `onMouseOver`.

**Comment by hadisfr** [comment]
It's something related to `calcLine`

https://github.com/aurora/flowdesigner/blob/1c9da16e1ffdc0f1b2c2b313c3d3dfc11737d819/src/wire.js#L29-L49

There is a small space between the pointer and the end of the path, too. I don't get what this function is expected to do actually, (creating this space?) but it seems there's no need to this function in `onMouseOver`.

**Comment by aurora** [comment]
Good question. I think i had an issue with event handling, when the end of the line was right below the mouse cursor and therefore added the gap, yes, calcLine is used to add the gap - i should have documented this. I will do some tests on this at the weekend.
